rtgui/controlspotpanel.cc

```diff
@@ -351,8 +355,10 @@ ControlSpotPanel::ControlSpotPanel():
     scopemask_->setAdjusterListener(this);
     denoichmask_->setAdjusterListener(this);
     transitBox->pack_start(*transit_);
-    transitBox->pack_start(*transitweak_);
-    transitBox->pack_start(*transitgrad_);
+  //  if(showsettings != 2) {
+        transitBox->pack_start(*transitweak_);
+        transitBox->pack_start(*transitgrad_);
+  //  }
```

Comment:
Do you want to keep these commented lines or is it ok to remove them?

Reply:
I remove